style.css

```diff
@@ -85,14 +86,41 @@ li {
   font-weight: 500;
 }
 
-#nav_menu {
+.hidden {
   display: none;
 }
 
-/* i added this to remove some white space that appeared
-after i put the burger menu inside a navbar */
+.overflow {
+  overflow: hidden;
+}
+
```

Comment:
- [OPTIONAL] Great job so far! 👏🏻  however according to the project requirement, 🤔  on the desktop size,  the closed button, and the menu mobile page must not be visible. Kindly correct this. You can use the `@media screen and (min-width: 768px) {`  to display none of your navigation mobile menu. 😃 

- Your design:

![18](https://user-images.githubusercontent.com/79058364/188732204-57ac65f8-2ced-4d7b-a1e1-7c2c497fc215.png)